app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -81,6 +81,28 @@
         closeSection($subsectionContent);
       }
 
+      function openLinkedSection() {
+        var anchor = getActiveAnchor(),
+            target,
+            section;
+
+        if (!anchor.length) {
+          return;
+        }
+
+        section = $element.find('#' + anchor)
```

Comment:
You could probably change this to `anchor` and negate the `substr` in the `getActiveAnchor` method below. Might be worth checking `.hash` always includes a # in all browsers, mind.